### PR TITLE
Prometheus

### PR DESCRIPTION
**Pull request by sbihel**
Only adds process info and requests count/latency.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -42,5 +42,7 @@ COPY --from=builder ${DIR}/target/x86_64-unknown-linux-musl/release/kepler /usr/
 COPY ./kepler.toml ./
 ENV ROCKET_ADDRESS=0.0.0.0
 EXPOSE 8000
+EXPOSE 8001
+EXPOSE 8081
```
Comment (cobward) [inline Dockerfile:46]: What's 8081 for?

**Review by cobward — APPROVED** [review]
LGTM!

**Comment by cobward** [comment]
Just thought, we might want to time the construction of the invoke wrapper too.

File: Dockerfile (hunk 1 above)
Comment (sbihel) [reply Dockerfile:46]: The relay, at least it's the default port in the config.

**Comment by sbihel** [comment]
> Just thought, we might want to time the construction of the invoke wrapper too.

I thought we would get this by subtracting the response time from Traefik but it's a bit involved. I'll add it.

**Pull request merged**